pkgs/by-name/py/pycdc/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pycdc";
+  version = "unstable-2024-10-13";
```

```suggestion
  version = "0-unstable-2024-10-13";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

---

can you also update commit message and PR title

---

Right. Fixed both.

---

@kirillrdy sorry from my end as well, I didn't notice that